mappings/net/minecraft/block/BlockState.mapping

```diff
@@ -3,11 +3,14 @@ CLASS bvj net/minecraft/block/BlockState
 		FIELD a DIRECTIONS [Lfa;
 		FIELD b cull Z
 		FIELD c fullOpaque Z
+		FIELD d translucent Z
+		FIELD e lightSubtracted I
```

Comment:
#384 is open for `darkness` instead of `lightSubtracted`. Since `lightSubtracted` is still used elsewhere, not sure what should be used here for now.

Reply:
I think it's fine for now, and rather be changed when the rest of the code is.